data/SI.slovenia/2014-2020/source.description.yaml

```diff
@@ -0,0 +1,145 @@
+
+    url: http://www.eu-skladi.si/sl/dokumenti/seznam-projektov/seznam-projektov-28-6-2016.xls
+
+    publication_date: 2016-09-16
+
+    constant_fields:
+      fund_name: ESRR, KS, ESS
+      offical_period: 2014-2020
+      currency_code: EUR
+
+
+      # any other field from fiscal.fields.yaml that is constant
+
+    schema:
+      fields: 
+
+        - name: Ime upravičenca/Beneficiary name
+          translates_to: Beneficiary name
+          maps_to: beneficiary_name
+
+        - name: Ime operacije/Operation name
+          translates_to: Operation name
+          maps_to: operation_name
+
+        - name: Opis operacije/Operation description
+          translates_to: Operation description
+          maps_to: operation_description
+                  
+        - name: Prednostna os
+          translates_to: Priority Axis
+          maps_to: priority_axis
+          
+        - name: Povzetek operacije/Operation summary
+          translates_to: Operation summary
+          maps_to: operation_summary    
+          
+        - name: Leto dodelitve sredstev/Year allocation
+          translates_to: Year allocation
+          maps_to: year_of_allocation          
+          
+        - name: Datum začetka/Operation start year
+          translates_to: Operation start year
+          maps_to: operation_start_year         
+
+        - name: Datum zaključka del/Operation end date
+          translates_to: Operation end date
+          maps_to: operation_end_date    
+          
+        - name: Skupni upravičeni izdatki/Total eligible expenditure allocated to the operation
+          translates_to: Total eligible expenditure allocated to the operation
+          maps_to: total_eligible_expenditure          
+          
+        - name: Odstotek sofinanciranja/union co-financing rate, as per priority axis
+          translates_to: union co-financing rate, as per priority axis
+          maps_to: eu_co_financing_rate                 
+          
+        - name: Naslov upravičenca/Beneficiary address
+          translates_to: Beneficiary address
+          maps_to: beneficiary_address
+          
+        - name: Občina/Municipality
+          translates_to: Municipality
+          maps_to: municipality      
+          
+        - name: Sklad/Fund
+          translates_to: Fund
+          maps_to: fund_name            
+          
+        - name: Regija/Region
+          translates_to: Region
+          maps_to: region      
+          
+        - name: Instrument/Instrument
+          translates_to: Instrument
+          maps_to: instrument 
+          
+        - name: Država/Country
+          translates_to: Country
+          maps_to: country      
+          
+        - name: Stanje na dan/Date of last update of the list of operations
+          translates_to: Date of last update of the list of operations
+          maps_to: date_of_last_update_of_list_of_operations
```

Comment:
@seegerseeger where did you find the all the `maps_to` fields? `region` `instrument`... are not in the list ;-) Feels like I'm living in a parallel universe